src/parsers/STATICPOOLparser.py

```diff
@@ -0,0 +1,149 @@
+""" 
+parsers/STATICPOOLparser.py 
+
+A parser for a static pool. 
+
+Needs functionality: 
+
+- Given set of observations, update pool and training sets
+    - update_pool_and_train()
+- Gets train and validation datasets 
+    - get_train_val_dataset() 
+
+"""
+
+from .base import Parser
+import torch
+import pandas as pd
+from common import data_split, CSVLoader, HDFLoader, PickleLoader, apply_scaler
+
+
+class STATICPOOLparser(Parser):
+    """This is a dummy class as for the moment the sampler handles everything"""
```

Comment:
Perhaps it would be useful to specify in the docstring that the usage intended is that the data should be stored as attributes and prior to scaling